lib/migrations/10-achievement-fields.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) => {
+      return Promise.all([
+        queryInterface.addColumn('achievement', 'icon_name', Sequelize.STRING, {transaction: t}),
+        queryInterface.addColumn('achievement', 'map_image', Sequelize.TEXT, {transaction: t}),
```

Comment:
I think this is better off as `Sequelize.STRING` for space reasons, sorry I wasn't clear in the previous comment.
Please change this one and the model file so that `map_image` has a type of `STRING`

Reply:
Revised